versions/development/SPEC.md

```diff
@@ -2112,6 +2112,7 @@ However, you can import two workflows with different namespace identifiers that
 Scopes are defined as:
 
 * `workflow {...}` blocks
+* `task {...}` blocks
 * `call` blocks
```

Comment:
Why is a call block a scope?
```
call x { Int a = 1 input: b = a }
```
It is not valid according to the Call Statement description, and I can't see the point
```

Reply:
I was wondering that too. Since `call` declares a bound name in the current scope, maybe that's the intent of including it here?